File: csharp/ex048.cs
// Faça um programa que leia 7 números inteiros e no final mostre o somatório
// entre eles.


class Ex048
{
  public int counter = 0;
  public int soma = 0;

  public void FirstCountdown()
  {
    while (counter <= 7)
    {
      Console.WriteLine("Digite um numero: ");
      int inputNumero = Convert.ToInt32(Console.ReadLine());

      soma += inputNumero;
    }
    Console.Write($"Soma : {soma}");
  }
}

File: python/ex048.py
# Faça um programa que leia 7 números inteiros e no final mostre o somatório
# entre eles.

counter = 0
inputNumber = 0
totalSum = 0

while counter <= 6:
    inputNumber = int(input("Digite um numero inteiro: "))
    totalSum+= inputNumber
    counter+=1
print(f"A soma de todos os numeros é {totalSum}")